genai-perf/genai_perf/record/types/gpu_power_limit_base.py

```diff
@@ -24,6 +25,7 @@ class GPUPowerLimitBase(IncreasingGPURecord):
     """
 
     base_tag = "gpu_power_limit"
+    reduction_factor = ReductionFactor.NONE
```

Comment:
why import no reduction factor?
is the reduction handled in a base class?
or is it for uniformity?

Reply:
This is the base class. Every base class needs a `base_tag` and a `reduction_factor`.  In the case of GPU Power Limit (as well as ISL/OSL, request throughput, etc...) we don't need to reduce the value. This both makes the function work and serves as documentation.

Reply:
Gotcha. I missed that in the file name.